docs/tests/common-setup.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+_common_setup() {
+    TEST_BREW_PREFIX="$(brew --prefix)"
+    load "${TEST_BREW_PREFIX}/lib/bats-support/load.bash"
+    load "${TEST_BREW_PREFIX}/lib/bats-assert/load.bash"
```

Comment:
I added `BATS_LIB_PATH` to make it work on Linux without `brew`, and people can set `BATS_LIB_PATH` if they have it installed somewhere else locally.

Found this technique here https://github.com/bats-core/homebrew-bats-core/blob/main/tests/tests_helper.bash#L1-L8

(Perhaps we can take advantage of this technique in add-ons, I will leave a note on this for myself.)

```suggestion
    TEST_BREW_PREFIX="$(brew --prefix 2>/dev/null || true)"
    export BATS_LIB_PATH="${BATS_LIB_PATH}:${TEST_BREW_PREFIX}/lib:/usr/lib/bats"
    bats_load_library bats-support
    bats_load_library bats-assert
```